src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,71 @@
+import React , { useState , useEffect } from 'react'
+import "./Homepage.css"
+import axios from "axios"
+import OnePlus from "../../images/OnePlus.jpg"
+import {Nav} from "../../components/Nav/Nav"
+import { Link } from "react-router-dom"
+
+function Homepage() {
+
+const [categories,setCategories] = useState([]);
+
+useEffect(() => {
+(async function fetchCategories() {
+try{
+const resp = await axios.get("/api/categories")
+console.log(resp);
+setCategories(resp.data.categories)
```

Comment:
code formatting seems not right. Running Prettier will help.

Reply:
Okay. I will have a look into it.